tests/scripts/sort_configs.py

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env python
+"""Script to sort conf/logs.json schema file"""
+import json
+import os
+import logging
+
+from collections import OrderedDict
+
+SCRIPT_NAME = 'JsonFileSorter'
+CONF_LOGS_FILE = os.path.join(os.path.dirname(__file__), '../../conf/logs.json')
+
+
+class JsonFileSorter(object):
+    """
+    Tests that the conf.json file is formatted properly and is sorted alphabetically
+    on the top-level key
+    """
+
+    def __init__(self):
+        self._logger = logging.getLogger(SCRIPT_NAME)
```

Comment:
since this is a standalone script (not a package), I'd suggest just using the `root` logger here:

`self._logger = logging.getLogger()`